### PR TITLE
Latest 0.7.0 (removing of sources), and istio lean

### PR DESCRIPTION
**Pull request by matzew**
patching w/ latest, and istio lean

@rhuss mind to double check ?

### CODE REVIEW

**Comment by rhuss** [comment]
Looks good, but changes to a fixed version instead of latest for eventing ?

**Comment by matzew** [comment]
> Looks good, but changes to a fixed version instead of latest for eventing ?

yeah - the official release (0.7.0) - instead of head

perhaps we could add a flag: if HEAD is wanted or noit ?

**Comment by rhuss** [comment]
I think something like `--latest` would make sense.

File: setup.sh [hunk 1]
```diff
@@ -67,7 +65,7 @@ header_text "Waiting for istio to become ready"
 sleep 5; while echo && kubectl get pods -n istio-system | grep -v -E "(Running|Completed|STATUS)"; do sleep 5; done
 
 header_text "Setting up Knative Serving"
-curl -L "https://raw.githubusercontent.com/openshift-knative/knative-serving-operator/master/deploy/resources/knative-serving-${serving_version}.yaml" \
+curl -L "https://github.com/knative/serving/releases/download/${serving_version}/serving.yaml" \
   | sed 's/LoadBalancer/NodePort/' \
   | kubectl apply --filename -
```
Comment (rhuss) [inline setup.sh:76]: I have something like 

```bash
set +e
kubectl apply --selector knative.dev/crd-install=true --filename https://storage.googleapis.com/knative-nightly/serving/latest/serving.yaml
rc=$?
set -e
if [ $rc -ne 0 ]; then
   # Retry
  sleep 5
  kubectl apply --selector knative.dev/crd-install=true --filename https://storage.googleapis.com/knative-nightly/serving/latest/serving.yaml
fi

curl -L "https://storage.googleapis.com/knative-nightly/serving/latest/serving.yaml" \
  | sed 's/LoadBalancer/NodePort/' \
  | kubectl apply --filename -
```

the URL need to be replaced, but this just retries to apply the CRDs if it fails the first time.

**Comment by matzew** [comment]
> I think something like --latest would make sense.

@rhuss OK, let's do that in a different PR

**Pull request merged**